src/Products/Annotation/Annotator/ArrowAnnotator.cs

```diff
@@ -1,82 +1,93 @@
+
+        public override AnnotationBase AnnotateWord()
         {
             withGuid = false;
-            AnnotationInfo arrowAnnotation = InitAnnotationInfo();
+            arrowAnnotation = InitAnnotationBase(arrowAnnotation) as ArrowAnnotation;
             return arrowAnnotation;
         }
-        
-        public override AnnotationInfo AnnotatePdf()
+
+        public override AnnotationBase AnnotatePdf()
         {
             withGuid = false;
-            AnnotationInfo arrowAnnotation = InitAnnotationInfo();
+            arrowAnnotation = InitAnnotationBase(arrowAnnotation) as ArrowAnnotation;
             return arrowAnnotation;
         }
-        
-        public override AnnotationInfo AnnotateCells()
+
+        public override AnnotationBase AnnotateCells()
         {
-            throw new NotSupportedException(String.Format(Message, annotationData.type));
+            throw new NotSupportedException(string.Format(Message, annotationData.type));
         }
-        
-        public override AnnotationInfo AnnotateSlides()
+
+        public override AnnotationBase AnnotateSlides()
         {
             withGuid = true;
-            // init annotation object
-            AnnotationInfo arrowAnnotation = InitAnnotationInfo();
-            arrowAnnotation.BackgroundColor = 15988609;
+            arrowAnnotation = InitAnnotationBase(arrowAnnotation) as ArrowAnnotation;
             return arrowAnnotation;
         }
-        
-        public override AnnotationInfo AnnotateImage()
+
+        public override AnnotationBase AnnotateImage()
         {
             withGuid = false;
-            // init annotation object
-            AnnotationInfo arrowAnnotation = InitAnnotationInfo();
-            arrowAnnotation.BackgroundColor = -15988609;
+            arrowAnnotation = InitAnnotationBase(arrowAnnotation) as ArrowAnnotation;
             return arrowAnnotation;
         }
 
-
-        public override AnnotationInfo AnnotateDiagram()
+        public override AnnotationBase AnnotateDiagram()
         {
             withGuid = false;
-            // init annotation object
-            AnnotationInfo arrowAnnotation = InitAnnotationInfo();
-            arrowAnnotation.BackgroundColor = 15988609;
+            arrowAnnotation = InitAnnotationBase(arrowAnnotation) as ArrowAnnotation;
             return arrowAnnotation;
         }
 
-
-        protected AnnotationReplyInfo getAnnotationReplyInfo(CommentsEntity comment)
+        protected override Reply GetAnnotationReplyInfo(CommentsEntity comment)
         {
-            AnnotationReplyInfo annotationReplyInfo = base.GetAnnotationReplyInfo(comment);
+            Reply annotationReplyInfo = base.GetAnnotationReplyInfo(comment);
             if (withGuid)
             {
-                annotationReplyInfo.ParentReplyGuid = annotationData.id.ToString();
+                annotationReplyInfo.ParentReply = new Reply() { Id = annotationData.id };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove these redundant parentheses.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372471)